packages/orchestration/src/types.js

```diff
@@ -1,5 +1,11 @@
 // @ts-check
+import '@agoric/network/exported.js';
 import '@agoric/vats/exported.js';
 import '@agoric/zoe/exported.js';
 
 export {};
+
+/**
+ * // XXX FIXME, network/src/types.js' is not a module
```

Comment:
https://github.com/Agoric/agoric-sdk/issues/4560 tracks this problem

Reply:
Thank you. Turns out this declaration is not necessary with `import '@agoric/network/exported.js';` so removing